user_docs/en/changes.t2t

```diff
@@ -4,21 +4,26 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.2 =
+This release introduces preliminary Windows 11 support.
+While Windows 11 is yet to be released, this release has been tested on preview versions of Windows 11.
+This includes an important fix for Screen Curtain (see important notes).
+The COM Registration Fixing Tool can now resolve more problems when running NVDA.
+There are updates to the synthesizer eSpeak and braille translator LibLouis.
+There are also various bug fixes and improvements, notably for braille support and Windows terminals, calculator, emoji panel and clipboard history.
+
 
 == Important notes ==
-- Screen Curtain on previous releases of NVDA is no longer compatible with versions of Windows newer than Windows 10 21H2 (10.0.19044).
+Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows newer than Windows 10 21H2 (10.0.19044).

```

Comment:
> newer than

Shouldn't this be "when using x or later"?

Reply:
Actually yes, that is the case as far as I'm aware. Before I fix this (across the rest of the documentation) I'm going to check this [via the release preview channel for WIN1021H2](https://blogs.windows.com/windows-insider/2021/07/15/announcing-windows-10-insider-preview-build-19044-1147-21h2/). I have confirmed the previous release of Windows works, Windows 10 21H1.

Reply:
The change of behaviour occurred in an insiders build, #12491 reports this change of behaviour for a build sometime after the `WIN10 21H2` build number. I am yet to be able to test `WIN10 21H2` preview (haven't been offered the update). It would be good if someone can confirm whether or not screen curtain works with the `WIN10 21H2` preview, but for now I will move forward to say "when using it or later" due to being uncertain.